tools/FakeItEasy.Build/FakeItEasy.Build.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Bullseye" Version="1.1.0-rc.2" />
+    <PackageReference Include="SimpleExec" Version="2.2.0" />
+    <PackageReference Include="GitVersion.CommandLine" Version="4.0.0-beta0012" ToolName="GitVersion" ToolExe="GitVersion.exe" />
+    <PackageReference Include="PdbGit" Version="3.0.41" ToolName="PdbGit" ToolExe="PdbGit.exe" />
+    <PackageReference Include="xunit.runner.console" Version="2.0.0" ToolName="XUnit" ToolExe="xunit.console.exe" />
+    <PackageReference Include="vswhere" Version="1.0.62" ToolName="VSWhere" ToolExe="vswhere.exe" />
+    <PackageReference Include="NuGet.CommandLine" Version="4.1.0" ToolName="NuGet" ToolExe="nuget.exe" />
+  </ItemGroup>
+
+  <Target Name="GenerateToolPaths" BeforeTargets="BeforeBuild">
+    <PropertyGroup>
+      <ClassName>ToolPaths</ClassName>
+      <OutputFile>$(ClassName).cs</OutputFile>
+    </PropertyGroup>
+    <ItemGroup>
+      <Tool Include="@(PackageReference)" Condition="'%(PackageReference.ToolName)' != ''" />
+      <Compile Remove="$(OutputFile)" />
+      <Compile Include="$(OutputFile)" />
+    </ItemGroup>
+    <PropertyGroup>
+      <Semicolon>%3B</Semicolon>
+      <Indent>%20%20%20%20%20%20%20%20</Indent>
+      <FileContent><![CDATA[namespace FakeItEasy.Build
+{
+    internal static class $(ClassName)
+    {
+@(Tool -> '$(Indent)public static string %(ToolName) = @"$(NuGetPackageRoot)%(Identity)\%(Version)\tools\%(ToolExe)"$(Semicolon)')

```

Comment:
const instead of static? Silences some warnings if you end up running StyleCop on the generated output, like I'm pushing for above.

Reply:
Oh! Yes, of course. For some reason I considered making it a method at some point, and forgot to change the static. I'll fix it.